hbase-client/src/main/java/org/apache/hadoop/hbase/client/MetaScanner.java

```diff
@@ -220,6 +220,7 @@ private static Result getClosestRowOrBefore(final Table metaTable, final TableNa
   throws IOException {
     byte[] searchRow = HRegionInfo.createRegionName(userTableName, row, HConstants.NINES, false);
     Scan scan = Scan.createGetClosestRowOrBeforeReverseScan(searchRow);
+    scan.addFamily(HConstants.CATALOG_FAMILY);
```

Comment:
The real error comes from ConnectionManager, but I tried to audit all reads against meta table and this was the only other one which did not have a family specified. I figured I should add it while I'm here even if we haven't hit an issue with this yet.